src/component/common/AnimateOnMount/AnimateOnMount.tsx

```diff
@@ -27,6 +31,7 @@ const AnimateOnMount: FC<IAnimateOnMountProps> = ({
         if (mountedRef.current !== mounted || mountedRef === null) {
             if (mounted) {
                 setShow(true);
+                onStart?.();
```

Comment:
Is this a special syntax for using the ? operator with a function? It looks to me like it's trying to access a property on the function.

Reply:
It's the optional chaining operator applied to a function: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Since onStart and onEnd are two optional props, this seems a bit cleaner than doing something like:
```JS
if (onStart) {
  onStart()
}
```